### PR TITLE
Footer - Social Media Links

### LINKED ISSUE #785: Footer - Social Media Links
In website footer, 4 seems the right number of social media links to show; I think we should add Instagram and delete either Mastodon or Github. Our Mastodon presence is very weak. Brylie, do you think it would be helpful / useful to leave Github link? Content is probably less accessible for common WF reader, but maybe highly appreciated by those who would be interested.

### PR DESCRIPTION
**Pull request by sim-codes**
#### Footer - Social Media Links #785
Closes #785 
#### Added two social media links (Instagram and Mastodon). Also did the ordering as specified by @marycklein.

### CODE REVIEW

File: .coveragerc [hunk 1]
```diff
@@ -4,3 +4,9 @@ omit =
     *tests*,
     *test*,
     core/*.py
+
+[run]
```
Comment (brylie) [inline .coveragerc:8]: It seems tat the ‘run’ section has already been defined in .coveragerc. This duplicate definition is causing errors in the CI script. Let’s merge your changes in to the existing ‘run’ definition so CI will pass

File: .coveragerc [hunk 2]
```diff
@@ -4,3 +4,9 @@ omit =
...
+plugins = django_coverage_plugin
+
+[django_coverage_plugin]
```
Comment (brylie) [inline .coveragerc:11]: Add newline
```suggestion
[django_coverage_plugin]

```

File: .coveragerc (hunk 1 above)
Comment (brylie) [reply .coveragerc:8]:
```suggestion
```

File: pyproject.toml [hunk 3]
```diff
@@ -141,3 +141,11 @@ target-version = "py311"
 
 [tool.setuptools]
 py-modules = []
+
+[tool.coverage.run]
+plugins = [
+    'django_coverage_plugin',
```
Comment (brylie) [inline pyproject.toml:147]: Do we need to add this dependency to our project dependencies?

Comment (sim-codes) [reply pyproject.toml:147]: Yes, install is available with pip.
I didn't want to tamper with the requirements.txt that's why I didn't include it

Comment (brylie) [reply pyproject.toml:147]: Go ahead and add it to the pyproject.toml requirements list and run `make compile-deps`, then commit the changes to requirements.txt files(s). It's a good improvement.

Also, make sure to pull the recent change from GitHub where I removed the duplicate `[run]` definition that was breaking CI.

Comment (brylie) [reply pyproject.toml:147]: Alternatively, remove the changes related to `django_coverage_plugin` and take that up in a separate task: https://github.com/WesternFriend/WF-website/issues/663

Comment (sim-codes) [reply pyproject.toml]: I just did that

**Review by brylie — APPROVED** [review]

**Comment by brylie** [comment]
@all-contributors please add @sim-codes for code

**Pull request merged**